packages/connect-node/src/connect-protocol.ts

```diff
@@ -139,6 +157,9 @@ export function createConnectProtocol(
             const type = connectParseContentType(
               req.headers["content-type"] ?? null
             );
+            const shouldRequireHeader =
+              options.requireConnectProtocolHeader ?? false;
+
```

Comment:
It looks like this variable isn't necessary, because it's already declared on line 73.

Reply:
good catch!